payment/src/test/java/org/killbill/billing/payment/api/TestPaymentApi.java

```diff
@@ -1446,7 +1446,7 @@ public void testVerifyJanitorFromPendingDuringCompletionFlow() throws PaymentApi
             final Payment completedPayment = createPayment(TransactionType.AUTHORIZE, initialPayment.getId(), initialPayment.getExternalKey(), transactionExternalKey, authAmount, PaymentPluginStatus.PROCESSED);
             Assert.fail();
         } catch (final PaymentApiException e) {
-            Assert.assertEquals(e.getCode(), ErrorCode.PAYMENT_INVALID_OPERATION.getCode());
+            Assert.assertEquals(e.getCode(), ErrorCode.PAYMENT_ACTIVE_TRANSACTION_KEY_EXISTS.getCode());
```

Comment:
@pierre this fixing will change the behavior of those 2 existing use cases (this one and the one right below). do you have an concerns about this?